src/java.base/share/classes/java/net/NetworkInterface.java

```diff
@@ -116,6 +141,11 @@ public Enumeration<InetAddress> getInetAddresses() {
      * {@link NetPermission}("getNetworkInformation") permission, then all
      * InetAddresses are returned.
      *
+     * @implNote
+     * The stream contains all or a subset of the InetAddresses that were
```

Comment:
perhaps additional comma for emphasis
The stream contains all, or a subset, of the InetAddresses . . .